permissions.go

```diff
@@ -368,6 +368,33 @@ func (jobState *JobState) getPermissionsForRunStep(step Step) ([]string, error)
 		}
 	}
 
+	// Dependabot auto approve
+	if step.Env["GITHUB_TOKEN"] != "" && isGitHubToken(step.Env["GITHUB_TOKEN"]) {
+		if strings.Contains(runStep, "gh pr review --approve") {
+			permissions = append(permissions, "pull-requests: write")
+			return permissions, nil
+		}
+	}
+
+	// Dependabot auto merge
```

Comment:
@Devils-Knight please add a comment with link/ reference to the action, like we do if logic is in the action-security.yml file.